### ISSUE TITLE
samclip not fix-mated

### ISSUE THREAD

**Issue by lskatz** [tseemann/shovill#139]
per this issue, it looks like samclip needs to be followed by `fixmate`.

https://github.com/tseemann/samclip/issues/7

**Comment by tseemann** [comment]
Yes I run `fixmate` in the Snippy pipeline for which this tool was developed.

I do need to clarify that in the docs.

**Comment by tseemann** [comment]
@lskatz i wonder if i should remove `samclip` or add `fixmate` - it has a larger overhead of double sorting.

**Comment by lskatz** [comment]
I'm honestly unsure. I appreciate the consideration for unnecessary sorting. Anyone out there know?
